ibexa/oss/4.5/assets/js/react.blocks.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
page-builder is not part of the OSS edition, it should be added to all relevant editions.